test/github/ds_test.rb

```diff
@@ -0,0 +1,22 @@
+require "test_helper"
+
+class GitHub::DSTest < Minitest::Test
+  def test_configures_correctly
+    GitHub::DS.configure do |config|
+      config.table_name = "example_key_values"
+    end
+
+    assert_equal GitHub::DS.config.table_name, "example_key_values"
+    GitHub::DS.reset
```

Comment:
Is there any place we could put this reset in addition to here to ensure it runs before every test?

Reply:
Placed it in appropriate `setup` methods!